### DISCUSSION TITLE
where is helpers.yt supposed to be located or installed?

### DISCUSSION THREAD

**Discussion by djrocket** [danielmiessler/Fabric#279] [Q&A]
Everything was working great until I did a git pull to get updates.  Now I'm getting this error after successfully re-running the setup and trying to execute yt summarize.  Before this pull the setup recognized that I use fabric_venv and installed everything properly into the venv.  But the new setup seems to put helpers.yt and ollama(?) somewhere else?  Any suggestions would be greatly appreciated:

yt --transcript https://youtu.be/****** | summarize

Traceback (most recent call last):
  File "/home/user/fabric/fabric_venv/bin/yt", line 3, in <module>
    from helpers.yt import main
ModuleNotFoundError: **No module named 'helpers.yt'**
Traceback (most recent call last):
  File "/home/user/fabric/fabric_venv/bin/fabric", line 6, in <module>
    sys.exit(cli())
             ^^^^^
  File "/home/user/fabric/installer/client/cli/fabric.py", line 101, in main
    standalone = Standalone(args, args.pattern)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/fabric/installer/client/cli/utils.py", line 54, in __init__
    sorted_gpt_models, ollamaList, claudeList = self.fetch_available_models()
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/fabric/installer/client/cli/utils.py", line 304, in fetch_available_models
    import ollama
ModuleNotFoundError: No module named 'ollama'

**Comment by nic0711** [comment]
Same problem here...  M2 MAC, LM Studio, Mistral, Conda Env

**Reply by djrocket** [reply]
the yt helper has the module import:
**from helpers.yt import main**

but I don't see helpers.yt not only in the local bin directory, the search can't find it anywhere

**Comment by cannuri** [comment]
I solved it by going to the fabric directory and running `poetry install` after pulling the latest updates
